webapi/src/test/scala/org/knora/webapi/it/v1/DrawingsGodsV1ITSpec.scala

```diff
@@ -89,7 +91,7 @@ class DrawingsGodsV1ITSpec
       // Upload the image to Sipi.
       val sipiUploadResponse: SipiUploadResponse = uploadToSipi(
         loginToken = loginToken,
-        filesToUpload = Seq(FileToUpload(path = pathToChlaus, mimeType = MediaTypes.`image/tiff`))
+        filesToUpload = Seq(FileToUpload(path = pathToChlaus, mimeType = org.apache.http.entity.ContentType.IMAGE_TIFF))
```

Comment:
what exactly is the benefit of this change? And can it be shortened to `ContentType.IMAGE_TIFF` with an import statement? That would look better IMO

Reply:
There is a namespace clash between `akka-http` and `Java`. This will be cleaned up as soon as we get rid of the Apache Http Client.